.circleci/config.yml

```diff
@@ -281,3 +291,4 @@ workflows:
           filters:
             branches:
               only: master
+      - javadoc_test
```

Comment:
Move this up above the it tests so that it's easier to see what runs on all branches.